src/AnswerInput.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+export default class AnswerInput extends Component {
```

Comment:
It's better to define a function rather than a class if you aren't tracking any local component state.